lib/content/gittenify.js

```diff
@@ -0,0 +1,33 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
+
+  for (const u of store.users) {
```

Comment:
hmm interesting, I guess since you are re-declaring it with a `const` keyword on every turn of that `for`-loop it is like you are just wiping it away and then initializing it fresh every time so that the variable is actually not being changed but erased and re-declared.